packages/entities/entities-key-sets/src/components/KeySetList.vue

```diff
@@ -61,33 +61,43 @@
         v-if="!filterQuery && config.app === 'konnect'"
         #empty-state
       >
-        <EntityEmptyState
-          :action-button-text="t('keySets.list.empty_state_v2.create_cta')"
-          appearance="secondary"
-          :can-create="() => canCreate()"
+        <KEmptyState
           data-testid="key-sets-entity-empty-state"
-          :description="t('keySets.list.empty_state_v2.description')"
-          :learn-more="config.app === 'konnect'"
+          icon-background
+          :message="t('keySets.list.empty_state_v2.description')"
           :title="t('keySets.list.empty_state_v2.title')"
-          @click:create="handleCreate"
-          @click:learn-more="$emit('click:learn-more')"
         >
           <template #image>
```

Comment:
We need to change this to the icon slot:
```suggestion
          <template #icon>
```

Otherwise the icon is aligned to the left:
<img width="2924" height="850" alt="image" src="https://github.com/user-attachments/assets/6bdc82ed-6e17-4da7-9217-124cf5448ace" />

Reply:
Ah good catch, thanks!